src/components/PassingTimesEditor/FixedPassingTimeEditor/FixedPassingTimeEditor.tsx

```diff
@@ -28,6 +28,7 @@ export const FixedPassingTimeEditor = ({
       <PassingTimePicker
         label={formatMessage({ id: 'passingTimesArrivalTime' })}
         disabled={index === 0}
+        ignoreSelectedTime={index === 0}
```

Comment:
This is when either arrival (1st passing time) or departure (last passing time) times are disabled but may still have a value coming from a stop point that previously had both of those but got reordered into the 1st or the last position in the stop point sequence